manifests/puppetmaster.pp

```diff
@@ -37,16 +37,19 @@
 
   if $reports {   # foreman reporter
 
-    exec { 'Create Puppet Reports dir':
-      command => "/bin/mkdir -p ${puppet_basedir}/reports",
-      creates => "${puppet_basedir}/reports",
-    }
-    file {"${puppet_basedir}/reports/foreman.rb":
-      mode    => '0644',
-      owner   => 'root',
-      group   => '0',
-      source  => "puppet:///modules/${module_name}/foreman-report_${report_api}.rb",
-      require => Exec['Create Puppet Reports dir'],
+    if ( $::foreman::params::aio_package == false ) {
```

Comment:
So how do we want to deal with this?
If this get's split out into it's own module this change will not be necessary.
If I migrate this to copy the file into the lib directory then it will be writing a file to a directory that get's synced from the server.